kytos/core/api_server.py

```diff
@@ -59,6 +59,24 @@ def register_rest_endpoint(self, url, function, methods):
             url = url[1:]
         self._start_endpoint(f'/kytos/{url}', function, methods=methods)
 
+    def remove_rest_endpoints(self, napp_id):
+        """Remove a rest endpoint from flask.
+
+        This method will remove all endpoints that contain the napp_id.
+
+        Args:
+            napp_id: String using the format '<username>/<napp_name>'
+
+        """
```

Comment:
```
Remove all decorated endpoints.

Args:
    napp (NApp): NApp instance to look for rest-decorated methods.
```
By receiving a NApp instance, you can code like `register_napp_endpoints` to get the same endpoints.